src/protocol/thread/pause.js

```diff
@@ -1,4 +1,4 @@
-const { sendMessage } = require("../socket");
+const { client } = require("../socket");
 const { defer, assert } = require("../utils");
 const { ValueFront } = require("./value");
```

Comment:
is there any reason not to import these as well?

Reply:
I just removed the unused import: the calls to `sendMessage()` were replaced by the corresponding client calls...